src/app/cli/src/init/coda_run.ml

```diff
@@ -690,6 +690,10 @@ let handle_shutdown ~monitor ~time_controller ~conf_dir ~child_pids ~top_logger
                    ~log_issue:false
                in
                Core.print_string message ; Deferred.unit
+           | Mina_lib.Offline_shutdown ->
+               Core.print_string "\nMINA DAEMON HAS BEEN OFFLINE FOR TOO LONG\nSHUTTING DOWN\n";
```

Comment:
Just my taste, but I'd prefer lowercase and surrounding with `***`, rather than SHOUTING.